CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to KoLmafia
+
+First off, thank you for contributing your time and knowledge. This project
+would not exist if not for volunteers like you!
+
+## Workflow Overview
+
+Some suggested reading (lifted from [Github docs](https://github.com/github/docs/blob/main/CONTRIBUTING.md)):
+
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+In the spirit of Github flow, all work must be done in feature branches
+(typically within your fork of the project), and reviewed by at least one of the
+core developers through Github's Pull Request (PR) feature.
+
+We strongly encourage that you provide tests as part of your pull request, as
+this helps us understand the problem at hand and avoid breaking your change in
+the future.
+
+## Guidelines (adapted from [our previous guide](https://kolmafia.us/threads/25364))
+
+Before creating a PR, we strongly recommend creating a bug in the [KoLmafia
+forums](https://kolmafia.us/forums/bug-reports.24/), in order to provide broader
+context regarding the change, and to discuss whether the change is a good
+idea. We'd hate for you to waste effort and be discouraged from submitting
+future PRs because some fundamental concerns arise during code review.
+
+The ability to determine the appropriateness of a patch and the level of support
+for it is extremely useful. A patch that violates the spirit of KoLmafia (for
+example, that allows access to non-KoLmafia directories in the user's file
+system) will be rejected. A patch that changes an existing feature will need
+some discussion. A patch that changes a lot of code to implement a feature of at
+best niche interest is unlikely to be accepted.
+
+The technical skills are the ability to write and test Java code that works in
+the context of KoLmafia. A prerequisite for success is the ability to read and
+understand KoLmafia's code and infer its architecture since those are not well
+documented in the traditional sense.
+
+Individuals with a good track record of creating PRs and constructively engaging
+in other pull requests can ask to be made a dev. The active devs who can do this
+are MCroft and gausie. We may also reach out to you.
```

Comment:
Is it still just us two?

Reply:
Depends on the definition. For expanding commit access, I think it's "whoever's listed as an owner in the github organization" which also includes Rinn and Veracity.